samples/python_interop/submit_qiskit_to_azure.ipynb

```diff
@@ -0,0 +1,255 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "90f546a9",
+   "metadata": {},
+   "source": [
+    "# Submitting Qiskit Circuits to Azure Quantum with the QDK\n",
+    "\n",
+    "This notebook shows how to take an arbitrary Qiskit `QuantumCircuit`, compile it to QIR using the Microsoft Quantum Development Kit (QDK) Python APIs, and submit it as a job to an Azure Quantum target. It also demonstrates how to handle parameterized circuits by binding parameters before submission."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2b838c23",
+   "metadata": {},
+   "source": [
+    "The workflow demonstrated here:\n",
+    "\n",
+    "1. Build (or load) a Qiskit `QuantumCircuit`.\n",
+    "2. Convert it to OpenQASM 3 text (`qiskit.qasm3.dumps`).\n",
+    "3. Compile that OpenQASM 3 source to QIR with `qdk.openqasm.compile`.\n",
+    "4. Create (or attach to) an Azure Quantum workspace with `qdk.azure.Workspace`.\n",
+    "5. Select a target (e.g. a simulator such as `rigetti.sim.qvm`).\n",
+    "6. Submit the QIR payload and retrieve measurement results.\n",
+    "\n",
+    "Why use the QDK in this flow?\n",
+    "\n",
+    "- Local compilation: QIR generation happens locally—no external CLI needed.\n",
+    "- Consistency: The same compiler stack powers Q#, OpenQASM, and (via translation) Qiskit interoperability.\n",
+    "- Extensibility: The same `compile` step can later drive resource estimation or optimization passes.\n",
```

Comment:
It's not quite "the same" compile step, because doing resource estimation you just call `qdk.openqasm.estimate` without using `compile` at all. It's more like the same "pattern" or "workflow." Also, I would avoid calling out optmization here, since that's something we don't currently offer through our tools.

Reply:
Ah okay, that makes sense. I've removed this line then since it seems it would cause more trouble than help.